lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -88,7 +104,22 @@ export class NativeInteractionClient extends BaseInteractionClient {
                     success: false,
                     errorCode: error.errorCode,
                     subErrorCode: error.subError,
-                    isNativeBroker: true
+                    isNativeBroker: true,
```

Comment:
Please ask Anthony to add an indicator as to whether the device is running on battery and adding that to the mats data they present and we log.  Their team did earlier research to indicate token requests were much slower when devices were on battery.

Reply:
Will add this once it's been added to the mats response